.github/src/EngineManager.java

```diff
@@ -1,15 +1,73 @@
 /*
- * represented by "Comp #1" in system diagram
+ * Assignment 4 instructions: 
+ * In the 'src' folder, implement the coordination component of the engine.
+ * This piece will do a couple of things, although exactly how it does these is
+ * up to you!
+ *      a.Receive requests from the user to start the computation (the
+ *        method signature for this will come from your API), and return a
+ *        suitable result status back to the user
+ * 
+ *      b. Request that the data storage component read integers from the
+ *      location specified by the user (remember, the number of integers
+ *      involved is potentially very large!)
+ * 
+ *      c. Pass the integers to the compute component
+ * 
+ *      d. Request that the data storage component write the results to the output
  */
 
+import java.util.List;
+
 public class EngineManager implements ComputeAPI {
+
     DataAPI dataStoreAPI;
     EngineAPI computeEngineAPI;
 
```

Comment:
Having the EngineManager constructor seems off. The User has an instance variable that has access to the methods inside of the "ComputeAPI" interface. 

The "ComputeAPI" interface requires that the EngineManager can handle the "compute" method that takes a ComputeRequest variable type.

I believe that the arguments in the "EngineManager" constructor should be empty.